qubesstats/__init__.py

```diff
@@ -44,9 +44,8 @@
 import stem.descriptor.reader
 
 LOGFILES = [
-    '/var/log/httpd-access.log',
-    '/var/log/httpd-access.log.0',
-    '/var/log/httpd-access.log.1',
+    '/var/log/nginx/access.log',
+    '/var/log/nginx/access.log.1',
```

Comment:
We need three files for the end of the month case: one, because log may be already rotated, second is the main log and third, because the rotation one month ago may have happened after midnight.

Is it the case that `.2` may actually be `.2.gz`?

Reply:
Yes, exactly. I don't know how to configure logrotate to keep last _two_ logs uncompressed. Alternatively, we may simply delete them instead of compressing? I.e. disable `compress` and set `keep 2`.

Reply:
Alternatively we may decompress those files (`import gzip; gzip.open(...)`). And handle missing files, so we actually don't have to hardcode if we are in BSD or in apache or in nginx. I think I'll do that instead. It will be easier to move the script again in the future.

Didn't we promise people we remove old logs anyway?

Reply:
> Didn't we promise people we remove old logs anyway?

Yes, in https://github.com/QubesOS/qubes-issues/issues/1624#issuecomment-171747034
I've adjusted the logrotate config so that 2.gz will not be compressed at all (it's only about that file in practice). We have about 60GB for that file, should be enough ;)

As for handling .gz, it looks unnecessary now. Missing files handling would be nice to have, but please don't spend more than 15min on this...